src/praisonai/praisonai/cli/commands/unified.py

```
@@ -0,0 +1,491 @@
+        finally:
+            sock.close()
+        
+        try:
+            # Create log directory for troubleshooting
+            log_dir = Path.home() / ".praisonai" / "unified" / "logs"
+            log_dir.mkdir(parents=True, exist_ok=True)
+            log_file = log_dir / f"{service}.log"
+            
+            if service == "flow":
+                # Use proper Typer entry point for flow
+                proc = subprocess.Popen([
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)